### DISCUSSION TITLE
Student account

### DISCUSSION THREAD

**Discussion by Ahmad-Alhindy** [community/community#142352] [GitHub Education] [answered]
Hi! I want to ask here about how to make your account ass student account I send what they asked to but that was not enouph so who should I speak with?

**Comment by LakinduVirajith** [comment, marked as answer]
If your GitHub student application wasn’t approved, double-check the email they sent for what might be missing. You can also contact GitHub’s support team directly https://support.github.com/contact/education for help.

If this helps, please mark it as the answer once your problem is solved!

Good luck!

**Comment by Oceanli12** [comment]
Thanks dear friends to help🙏I'm really new old man try to understand what going on,but I think come to me not easy I'm so lucky have so many specialist to help out 🤗🙏
